packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -203,10 +206,15 @@ class BlockListBlock extends Component {
 		return (
 			<TouchableWithoutFeedback
 				onPress={ this.onFocus }
+				hitSlop={ {
```

Comment:
Just to understand better: we already have a custom hitSlop in "cellRenderer", why was this one necessary?

Reply:
The job of the `cellRenderer` `hitSlop` is to make sure the touch goes to this control before the sibling that is being obscured. Then this `hitSlop` allows that touch to fall into this view's child controls where the Overflow view can then pick up the touch and perform its logic.